lib/handlers/ntlm.ts

```diff
@@ -146,12 +128,30 @@ export class NtlmCredentialHandler implements ifm.IRequestHandler {
             throw new Error('www-authenticate not found on response of second request');
```

Comment:
Can you change the comment at the top of this function to point to the new library we're using?

Reply:
Well, looking into [SamDecrock httpntlm module](https://github.com/SamDecrock/node-http-ntlm/blob/master/httpntlm.js#L41-L59), This function is indeed an adaptation of the original implementation, it is **NOT** of the newly used module, no ?

Reply:
Yeah, I guess you're right it still probably is. My mistake!